doc/sphinx/examples/index.md

```diff
@@ -0,0 +1,24 @@
+# Examples
+
+Cantera includes examples for a number of applications. To see all examples written in a
+particular programming language, select the corresponding category below. Or, to browse
+all examples covering a particular topic, regardless of programming language, select
+from the {ref}`list of example tags <tagoverview>`.
+
+% The following pages are generated by sphinx-gallery
```

Comment:
Is this a comment format for md?

Reply:
Yes, for MyST-flavored Markdown, at least.